exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.135.xml

```diff
@@ -0,0 +1,69 @@
+<entry>
+  <act classCode="ACT" moodCode="EVN">
+    <!-- Conforms to C-CDA 2.1 Problem Concern Act (V3) -->
+    <templateId root="2.16.840.1.113883.10.20.22.4.3" extension="2015-08-01" />
+    <!-- Diagnosis Concern Act -->
+    <templateId root="2.16.840.1.113883.10.20.24.3.137" />
+    <id root="{{newRandom}}" />
+    <code code="CONC" codeSystem="2.16.840.1.113883.5.6" displayName="Concern" />
+    <statusCode code="active" />
+    <effectiveTime>
+      <low {{valueOrNullFlavor .EntrySection.Entry.StartTime}} />
+    </effectiveTime>
+    <entryRelationship typeCode="SUBJ">
+      <observation classCode="OBS" moodCode="EVN">
+        <!-- Conforms to C-CDA R2.1 Problem Observation (V3) -->
+        <templateId root="2.16.840.1.113883.10.20.22.4.4" extension="2015-08-01" />
+        <!-- Diagnosis template -->
+        <templateId root="2.16.840.1.113883.10.20.24.3.135" />
+        <id root="1.3.6.1.4.1.115" extension="{{.EntrySection.Entry.ObjectIdentifier.ID}}" />
+        <code code="29308-4" codeSystem="2.16.840.1.113883.6.1">
+          <translation code="282291009" codeSystem="2.16.840.1.113883.6.96"/>
+        </code>
+        <statusCode code="completed" />
+        <effectiveTime>
+          <!-- Attribute: onset datetime -->
+          <low {{valueOrNullFlavor .EntrySection.Entry.StartTime}} />
+          <!-- Attribute: abatement datetime -->
+          <high {{valueOrNullFlavor .EntrySection.Entry.EndTime}} />
+        </effectiveTime>
+        {{template "_code.xml" codeDisplayWithPreferredCode .EntrySection.Entry .EntrySection.Entry.Coded .MapDataCriteria "entryCode"}}
+        {{if hasLaterality .EntrySection.Laterality}}
+          <targetSiteCode code="{{.EntrySection.Laterality.Code}}"
+                          codeSystem="{{oidForCodeSystem .EntrySection.Laterality.CodeSystem}}"
+                          {{if .EntrySection.Laterality.Title}}
+                            displayName="{{.EntrySection.Laterality.Title}}"
+                          {{end}}
+                          sdtc:valueSet="{{oidForCode .EntrySection.Laterality.CodedConcept (index .MapDataCriteria.FieldOids "LATERALITY")}}"
+          />
+        {{end}}
+        {{if hasAnatomicalLocation .EntrySection.AnatomicalLocation}}
+          <targetSiteCode code="{{.EntrySection.AnatomicalLocation.Code}}"
```

Comment:
Why? looking at the IG it seems that it's the `<value>` element that needs `xsi:type="CD"`